l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -64,12 +67,12 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
         address _l1LegacyBridge,
         bytes32 _l2TokenProxyBytecodeHash,
         address _aliasedOwner
-    ) external reinitializer(2) {
+    ) external initializer { // Why do we use reinitializer here insted of initializer?
```

Comment:
Just of curiosity

Reply:
See this:
https://github.com/matter-labs/era-contracts/pull/478/files#r1604948571
We are upgrading previous contract

Reply:
Ah okay :)